org.eclipse.lemminx/src/main/java/org/eclipse/lemminx/services/extensions/IDocumentLifecycleParticipant.java

```diff
@@ -0,0 +1,54 @@
+/*******************************************************************************
+* Copyright (c) 2021 Red Hat Inc. and others.
+* All rights reserved. This program and the accompanying materials
+* which accompanies this distribution, and is available at
+* http://www.eclipse.org/legal/epl-v20.html
+*
+* SPDX-License-Identifier: EPL-2.0
+*
+* Contributors:
+*     Red Hat Inc. - initial API and implementation
+*******************************************************************************/
+package org.eclipse.lemminx.services.extensions;
+
+import java.util.List;
+
+import org.eclipse.lemminx.dom.DOMDocument;
+import org.eclipse.lsp4j.TextDocumentContentChangeEvent;
+
+/**
+ * Document LifecycleService participant API.
+ * 
+ * @since 0.18.0
+ */
+public interface IDocumentLifecycleParticipant {
+
+	/**
+	 * Handler called when a XML document is opened.

```

Comment:
Very minor issue. It should be "... when a**n** XML ..."

Reply:
fixed